library/Zend/Code/Reflection/ClassReflection.php

```diff
@@ -177,12 +177,17 @@ public function getMethods($filter = -1)
         return $methods;
     }
 
+    /**
+     * Returns an array of reflection classes of traits used by this class.
+     *
+     * @return array|null
+     */
     public function getTraits()
     {
         $vals = array();
         $traits = parent::getTraits();
-        if (! $traits) {
-            return;
+        if ($traits === null) {
```

Comment:
if `$traits` is empty array then `if (! $traits)` is `true` and the return of `getTraits()` is `null`. It is incorrect behaviour which broke a lot of unittests in my projects.